src/satellite_consumer/process.py

```diff
@@ -152,7 +152,7 @@ def _serialize(d: dict[str, Any]) -> dict[str, Any]:
     if "time" not in da.dims:
         time = pd.to_datetime(rounded_time)
         da = da.assign_coords({"time": time}).expand_dims("time")
-
+    da.attrs.pop("_FillValue", None)
```

Comment:
im not sure this is right? I saw some errors like this with the upgrade from zarr2 to zarr3. What error do you get if you dont include this?

Reply:
If this isn't here, it gives an error of 

```
16:58:40.876 ERROR   [main.py:119] | Error: failed to prevent overwriting existing key _FillValue in attrs. This is probably an encoding field used by xarray to describe how a variable is serialized. To proceed, remove this key from the variable's attributes manually. 
```

Removing the FillValue here fixes that and allows for the dataarrays to be added to the Icechunk store successfully

Reply:
a funny, does that happen for you on the other satellites? 

What python version you running? What zarr, xarray and icechunk version you running?